src/content/comparison.md

```diff
@@ -16,36 +17,36 @@ related:
-| Multi pages build with common bundle | with manual configuration | **yes** | with manual configuration | with bundle arithmetic | no | no|
-| Multiple bundles | **yes** | with manual configuration | with manual configuration | yes | no | yes |
-| Node.js built-in libs `require("path")` | **yes** | no | **yes** | **yes** | [node-resolve-plugin](https://github.com/rollup/rollup-plugin-node-resolve) | |
-| Other Node.js stuff | process, __dir/filename, global | - | process, __dir/filename, global | process, __dir/filename, global for cjs | global ([commonjs-plugin](https://github.com/rollup/rollup-plugin-commonjs)) | |
-| Plugins | **yes** | yes | **yes** | yes | yes | yes |
-| Preprocessing | **loaders, [transforms](https://github.com/webpack-contrib/transform-loader)** | loaders | transforms | plugin translate | plugin transforms | compilers, optimizers |
-| Replacement for browser | `web_modules`, `.web.js`, package.json field, alias config option | alias option | package.json field, alias option | package.json, alias option | no | |
-| Requirable files | file system | **web** | file system | through plugins | file system or through plugins | file system |
-| Runtime overhead | **243B + 20B per module + 4B per dependency** | 14.7kB + 0B per module + (3B + X) per dependency | 415B + 25B per module + (6B + 2X) per dependency | 5.5kB for self-executing bundles, 38kB for full loader and polyfill, 0 plain modules, 293B CJS, 139B ES2015 System.register before gzip | **none for ES2015 modules** (other formats may have) | |
-| Watch mode | yes | not required | [watchify](https://github.com/browserify/watchify) | not needed in dev | [rollup-watch](https://github.com/rollup/rollup-watch) | yes |
+| Feature                                                              | webpack/webpack                                                                | jrburke/requirejs                                | substack/node-browserify                            | jspm/jspm-cli                                                                                                                                     | rollup/rollup                                                                | brunch/brunch                                                                              |
+| -------------------------------------------------------------------- | ------------------------------------------------------------------------------ | ------------------------------------------------ | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
```

Comment:
why table changes? please provide motivation or revert

Reply:
I guess prettier or IDE formated that automatically. It’s more readable in the source code but I will revert later today. Shall we format all tables like this? Maybe in another PR?

Reply:
Unfortunately this is not in `.editorconfig` so lets not change formatting of the tables for now.